src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -631,9 +639,10 @@ that the type of arrows in a Segal type is itself a Segal type.
   (A : U)
   (triangle : Δ² -> A)
   : Δ¹×Δ¹ -> A
-  := \ (t , s) ->                   -- two copies of the triangle along the common diagonal edge.
-    recOR (t <= s |-> triangle (s , t) ,
-        s <= t |-> triangle (t , s))
+  :=
+    \ (t , s) ->
+    recOR ( t <= s |-> triangle (s , t) ,
+            s <= t |-> triangle (t , s))
```

Comment:
Again, when breaking lines, always start at the earliest possible point to avoid excessive indentation. Here:
```rzk
    recOR
      ( t <= s |-> triangle (s , t) ,
        s <= t |-> triangle (t , s))
```